src/test/java/org/cactoos/text/TextOfTest.java

```diff
@@ -236,4 +238,51 @@ public void printsStackTrace() {
             )
         );
     }
+
+    @Test
+    public void readsFromInputStream() throws Exception {
+        final String before = "line1";
```

Comment:
@iCok I think 'content' would be a better mame for this variable, since it is not altered at all, it is just the read content.

Reply:
Same for the next 2 tests